js/pages/samples.js

```diff
@@ -588,11 +614,22 @@ const renderLocations = (site) => {
                         </div>
                     </div>
                 </div>`
-            } else {
+            } else if(site==u_chicago && !location[1]){
+                template += `<div class="row" style="width:100%">
+                <div style="width:100%">
+                    <div class="messagesHeaderFont">
+                    </div>
+                    <div class="messagesBodyFont">
+                        ${location[1]}
+                    </div>
+                </div>
+            </div>`
+            }
+            else{
```

Comment:
```suggestion
            } else {
```

Formatting suggestion